packages/css/src/__tests__/transform.test.ts

```diff
@@ -439,4 +435,44 @@ describe('#css-transform', () => {
       `);
     });
   });
+
+  it('should add extra specificity after atomicizing without affecting class names', () => {
+    const styles = `
+      padding: 8px;
+      color: red;
+      :before {
```

Comment:
would it be worthwhile to check selectors with `&` at the front as well? (e.g. `&::before`)

also nit: `/:before/::before/g`